platforms/documentation/docs/src/docs/userguide/authoring-builds/plugins/testing_gradle_plugins.adoc

```diff
@@ -313,6 +302,53 @@ TestKit determines the plugin classpath by running a specific Gradle task.
 You will need to execute the `assemble` task to initially generate the plugin classpath or to reflect changes to it even when running TestKit-based functional tests from the IDE.
 
 Some IDEs provide a convenience option to delegate the "test classpath generation and execution" to the build.
-In IntelliJ you can find this option under _Preferences... > Build, Execution, Deployment > Build Tools > Gradle > Runner > Delegate IDE build/run actions to gradle_.
+In IntelliJ, you can find this option under Preferences... > Build, Execution, Deployment > Build Tools > Gradle > Runner > Delegate IDE build/run actions to Gradle.
 
 image::intellij-delegate-to-build.png[]
+
+[[sec:writing_tests_for_your_plugin]]
+== Writing tests for your plugin
```

Comment:
💭 This section seems to be about writing unit tests for your Plugin class.  I would order it ahead of the integration/functional tests.

I would also have the type of test appear in the Contents, so that readers can quickly navigate to information about the type of test they want to write.

Reply:
Copy paste error.